applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -50,6 +50,10 @@ Complete the following steps to configure the module:
    Those options affect :ref:`nrf_desktop_usb_state` that subscribes for HID boot reports.
    The Dongle forwards HID reports from both mouse and keyboard, and so either option works if you want to have the Dongle work as boot mouse or boot keyboard.
    For more information about the configuration of the HID boot protocol, see the boot protocol configuration section in the :ref:`nrf_desktop_usb_state` documentation.
+#. nRF Desktop dongle can use one or more instances of the USB HID class.
```

Comment:
The whole sequence of steps here does not really seem to me as clear steps. The steps should be chrystal clear, always starting with an imperative word. After that, you can always spice up the step with additional and necessary information.

Reply:
You are right, but some steps are optional/conditional and might require introducing additional context before informing about the needed configuration update. Still, I think it's good to have them on the list to ensure users would not overlook the important configuration checks (we already follow this approach in multiple modules' docs). We also try to highlight some of the important configuration changes that are automatically applied under the hood.